packages/sonarwhal/src/lib/utils/caniuse.ts

```diff
@@ -0,0 +1,6 @@
+/*
+ * This file is necessary in order to avoid version problems runing browserlists.
+ */
+import * as caniuse from 'caniuse-api';
```

Comment:
Can you just import `isSupported`? This will help us with 🌳 shaking with webpack.

Reply:
I'll take care of that before merging.